### PR TITLE
 layout_stars

### PR DESCRIPTION
**Pull request by SweeeD1**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,54 @@
...
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0 stars">
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
+        <div class="stars__star none_active"> </div>
```
Comment (Anastasiia-Svintsova) [inline src/index.html:17]: Don't add additional class names to the `stars__star`. It's redundant. You can use a modifier for the stars block and  :nth() child for it
```suggestion
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
        <div class="stars__star"> </div>
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.active {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-size: 13px;
+  background-repeat: no-repeat;
+}
```
Comment (Anastasiia-Svintsova) [inline src/style.css:39]: As I said before, use a modifier for the stars block and :nth() child instead of additional classnames

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- All tests failed 😢 Fix it, please. If you have any questions, feel free to ask them in the chat 
image

File: readme.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://SweeeD1.github.io/layout_stars/)
+- [TEST REPORT LINK](https://SweeeD1.github.io/layout_stars/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:4]: Add these links as comment here 
image

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:12]: If we have no content, we can use a single tag, but this is optional
```suggestion
      <div class="stars__star" />
```

**Pull request closed without merging**